packages/world/src/interfaces/IWorldCore.sol

```diff
@@ -8,8 +8,8 @@ import { ISystemHook } from "./ISystemHook.sol";
 import { IModule } from "./IModule.sol";
 
 /**
- * The IWorldCore interfaces includes all World methods with a static function selector.
- * Consumers should use the `IBaseWorld` interface instead, which includes
+ * The IWorldCore interface only includes core World methods with a static function selector.
+ * Consumers should use the `IBaseWorld` interface instead, which includes other static and
```

Comment:
`IWorldCore` has all the "static" function selectors (those that are part of the World bytecode), while `IBaseWorld` also includes function selectors that are registered dynamically (those that are part of the bytecode of another contract but are called via the World's fallback function)
```suggestion
 * Consumers should use the `IBaseWorld` interface instead, which includes
```

Reply:
by "other static" I meant DynamicPartial funcs, but yea they are actually dynamic too